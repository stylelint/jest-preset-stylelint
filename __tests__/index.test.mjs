import preset from '../jest-preset.js';

it('should contain expected keys', () => {
	expect(Object.keys(preset)).toMatchSnapshot();
});
