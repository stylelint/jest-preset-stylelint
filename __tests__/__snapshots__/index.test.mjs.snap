// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`should contain expected keys 1`] = `
[
  "setupFiles",
  "setupFilesAfterEnv",
  "testEnvironment",
]
`;
